runner/cf-driver/prepare.sh

```diff
@@ -80,6 +80,15 @@ start_container () {
         --docker-image "$image_name"
     )
 
+    # Entrypoint & command aren't available w/o loading job res file
+    img_data=$(jq -rc '.image' "$JOB_RESPONSE_FILE")
+    container_entrypoint=$(echo "$img_data" | jq -r '.entrypoint | select(.)')
+    container_command=$(echo "$img_data" | jq -r '.command | select(.)')
```

Comment:
[question] what does `select(.)` do here? I would have thought it's basically a no-op

Reply:
It filters `null`. You also can't do `select(.command)` because `select` only returns its entire input (i.e., all the JSON) or nothing.

I'm sure there's more than a couple way to deal with nulls. It could error for `null`, or do `jq -r '.command // ""'` to get an empty string, probably other things.

Reply:
ah, cool - that seems as good a way to do that as any other I've seen.